src/index.html

```diff
@@ -11,6 +11,10 @@
       rel="stylesheet"
       href="./styles/main.scss"
     />
+    <script
+      src="scripts\main.js"
+      defer
+    ></script>
```

Comment:
The script tag is included twice in the document (lines 14-17 and line 123). It's redundant to include the same script twice. Consider removing one of them.